src/window.c

```diff
@@ -1327,6 +1333,11 @@ win_split_ins(
 	msg_col = 0;	// put position back at start of line
     }
 
+    // If we have there is a winbar, win_equal invalidates w_botline
+    // before we get to spsc_correct_scroll (why only with winbar?).
+    if (!p_spsc && WINBAR_HEIGHT(curwin))
```

Comment:
@brammool with the latest commit we also avoid scrolling in all cases when a winbar is present. Without this check `w_botline` was invalidated due to some recursive calls to `win_new_width()` in `equal_win()`, before we get to `spsc_correct_scroll()`. Resulting in scrolling in `curwin`(actually `prevwin` after `win_enter_ext(wp)`.

Would a check like this be okay? It's not really checking against the root cause of what causes scrolling but it seems to work and I don't see any side effects.